l10n_it_delivery_note/tests/test_stock_delivery_note_invoicing.py

```diff
@@ -1509,3 +1509,31 @@ def test_partial_invoicing_multiple_so(self):
         self.assertEqual(invoice_line.display_type, "line_note")
         self.assertEqual(invoice_line.quantity, 0)
         self.assertEqual(invoice_line.delivery_note_id, second_delivery_note)
+
+    def test_delivery_note_to_draft_from_create(self):
+        """
+        Create delivery_note from picking with already invoiced sale order then validate
+        and reset the status to draft.
+        """
+        sales_order = self.create_sales_order(
+            [
+                self.desk_combination_line,
+            ]
+        )
+        sales_order.action_confirm()
+        picking = sales_order.picking_ids
+        picking.move_lines[0].quantity_done = 1
+        picking.button_validate()
+        sales_order._create_invoices(final=False)
```

Comment:
Considerato che il default di `final` è già `False` (rif. https://github.com/odoo/odoo/blob/8355c10676e1dbb668702337863218bdda7c23dc/addons/sale/models/sale.py#L675) lo puoi eliminare. Potresti cogliere l'occasione per eliminarlo anche negli altri posti dive figura in questo file?

Reply:
fatto, grazie